apps/mochi-web/components/Feed/index.tsx

```diff
@@ -142,70 +111,18 @@ export default function Feed({ className = '' }: Props) {
           /* }} */
           className="overflow-hidden h-screen max-h-full"
         >
-          <div className="px-8 mx-auto" style={{ width: 1440 }}>
-            <div className="flex sticky top-0 z-10 flex-1 bg-feed-bg">
-              {[
-                'issued by',
-                '',
-                'recipients',
-                /* 'method', */
-                'amount',
-                'where',
-                'tx id',
-                'wen',
-                'status',
-              ].map((column, i) => {
-                return (
-                  <HeaderCell
-                    className={i === 7 ? 'pl-6' : ''}
-                    key={`feed-column-${column}`}
-                    width={colWidth[i]}
-                  >
-                    {i === 1 ? <div className="w-5" /> : column}
-                  </HeaderCell>
-                )
-              })}
-              <div
-                className={clsx(
-                  'transition top-full left-0 absolute w-full h-20 bg-gradient-to-b pointer-events-none from-feed-bg',
-                  {
-                    'opacity-100': isShowTopFade,
-                    'opacity-0': !isShowTopFade,
+          <div className="px-6 mx-auto" style={{ width: 1440 }}>
+            <TransactionTable
```

Comment:
I think the width of the table is not wide as the original version. Can you double check?

Reply:
This is in production -- same 1440:

<img width="1502" alt="image" src="https://github.com/consolelabs/mochi-ui/assets/11691985/c21f0b74-258e-457f-ba70-81e603362e39">